src/vg/civcraft/mc/citadel/Utility.java

```diff
@@ -539,22 +552,19 @@ public static boolean reinforcementBroken(Player player, Reinforcement reinforce
 			return false;
 		}
     	StringBuffer slb = null;
-		if (CitadelConfigManager.shouldLogBreaks()) {
+		boolean logIt = CitadelConfigManager.shouldLogFriendlyBreaks() && player != null;
+		if (logIt) {
 			slb = new StringBuffer();
-			if (player != null) {
-				slb.append("Player ").append(player.getName()).append(" [").append(player.getUniqueId())
-						.append("]");
-			} else {
-				slb.append("Something ");
-			}
-			slb.append("broke a ").append(reinforcement.getMaterial()).append(" reinforcement at ")
+			slb.append("Player ").append(player.getName()).append(" [").append(player.getUniqueId())
```

Comment:
I would use a different wording here so you can tell the difference between hostile and friendly breaks from the logs